Scripts/eyeball.js

```diff
@@ -34,47 +19,32 @@ $(function() {
 			heightWidthSetter = mouseYPosition;
 		}
 		if (heightWidthSetter < 50) {
-		//	$('#iris').css('height', (heightWidthSetter + 50) / 100  * irisSize + "px");
-		//	$('#irisBG').css('height', (heightWidthSetter + 50) / 100  * irisSize + "px");
-		//	$('#pupil').css('height', (heightWidthSetter + 50) / 100  * pupilSize + "px");
 			$('#iris').css('height', (.66 * irisSize) + (.34/50 * heightWidthSetter * irisSize) + "px");
-			$('#irisBG').css('height', (.66 * irisSize) + (.34/50 * heightWidthSetter * irisSize) + "px");
 			$('#pupil').css('height', (.66 * pupilSize) + (.34/50 * heightWidthSetter * pupilSize)  + "px");
 		} else {
-		//	$('#iris').css('height', ((heightWidthSetter - 150) / -1 / 100 * irisSize) + "px");
-		//	$('#irisBG').css('height', ((heightWidthSetter - 150) / -1 / 100 * irisSize) + "px");
-		//	$('#pupil').css('height', ((heightWidthSetter - 150) / -1 / 100 * pupilSize) + "px");
 			$('#iris').css('height', (1.34 * irisSize - (.34/50 * heightWidthSetter * irisSize)) + "px");
-			$('#irisBG').css('height', (1.34 * irisSize - (.34/50 * heightWidthSetter * irisSize)) + "px");
 			$('#pupil').css('height', (1.34 * pupilSize - (.34/50 * heightWidthSetter * pupilSize)) + "px");
 		}
 		// change shape for the iris and pupil
 		$('#iris').css('border-radius', '75%');
-		$('#irisBG').css('border-radius', '75%');
 		$('#pupil').css('border-radius', '75%');
 
 		//code for rotation
 		if (mouseYPosition <= 50) {
 			if (mouseXPosition < 50) {
 				$('#iris').css({'transform' : 'rotate(' + (270 + angle) + 'deg)'});
-				$('#irisBG').css({'transform' : 'rotate(' + (270 + angle) + 'deg)'});
 			}	else {
 				$('#iris').css({'transform' : 'rotate(' + (90 + angle) + 'deg)'});
-				$('#irisBG').css({'transform' : 'rotate(' + (90 + angle) + 'deg)'});
 			}
 		} else {
 			if (mouseXPosition < 50) {
 				$('#iris').css({'transform' : 'rotate(' + (270 + angle) + 'deg)'});
-				$('#irisBG').css({'transform' : 'rotate(' + (270 + angle) + 'deg)'});
 			}	else {
 				$('#iris').css({'transform' : 'rotate(' + (90 + angle) + 'deg)'});
-				$('#irisBG').css({'transform' : 'rotate(' + (90 + angle) + 'deg)'});
 			}
 		}
 		//code for cursor position and eyeball position
 	  $('#iris').css('left', 'calc(' + mouseXPosition + '% + ' + (-14 - 53/100 * mouseXPosition) + 'px)');
-	  $('#irisBG').css('left', 'calc(' + mouseXPosition + '% + ' + (-14 - 53/100 * mouseXPosition) + 'px)');
 		$('#iris').css('top', 'calc((' + mouseYPosition + '%) - (' + mouseYPosition / 100 * $('#iris').css('height').replace('px', "") + 'px)');
-		$('#irisBG').css('top', 'calc((' + mouseYPosition + '%) - (' + mouseYPosition / 100 * $('#iris').css('height').replace('px', "") + 'px)');
 	});
 });
```

Comment:
Wow, crazy eyeball! Did you write all of this yourself? If so, congratulations, that's impressive. If you got some or all of this from somewhere else, we need to have attribution in the comments or the README or something like that. It doesn't seem to work right in the Safari browser (eye always points up).